File: fincapp-backend/src/FincApp.API/DTOs/BulkSyncRequestDto.cs
using System;
using System.Text.Json.Serialization;

namespace FincApp.API.DTOs;

public class BulkSyncRequestDto
{
    [JsonPropertyName("device_uuid")]
    public string DeviceUuid { get; set; } = string.Empty;

    [JsonPropertyName("user_id")]
    public Guid UserId { get; set; }

    [JsonPropertyName("payload_mutations")]
    public PayloadMutationsDto PayloadMutations { get; set; } = new();
}

File: fincapp-frontend/mobile-app/app/src/main/java/com/irwi/fincapp/network/dto/BulkSyncRequestDto.java
package com.irwi.fincapp.network.dto;

import com.google.gson.annotations.SerializedName;

public class BulkSyncRequestDto {
    @SerializedName("device_uuid") public String deviceUuid;
    @SerializedName("user_id") public String userId;
    @SerializedName("payload_mutations") public PayloadMutationsDto payloadMutations;
}
